utils.ts

```diff
@@ -153,9 +153,9 @@ const filterOutDuplicateStops = (
   if (
     customFeatures.find(
       (otherFeature: Feature) =>
-        feature?.properties?.name
+        (feature?.properties?.name || '')
           .toLowerCase()
-          .includes(otherFeature?.properties?.name.toLowerCase()) ||
+          .includes((otherFeature?.properties?.name || '').toLowerCase()) ||
```

Comment:
I won't block this PR based on this since it was "broken" before, but I think that we should be more careful about using optional chaining to solve problems when our types aren't correct.